### DISCUSSION TITLE
Help Needed: How to Attend the GitHub Foundation Exam via the GitHub Student Developer Pack?

### DISCUSSION THREAD

**Discussion by md-ashraful-islam-ullas** [community/community#146455] [GitHub Education] [answered]
### Hi everyone,

I recently booked the GitHub Foundation exam for free through the GitHub Student Developer Pack. However, I’m unsure about where and how to attend the exam. Could someone guide me through the process or point me to the right resources?

Any help would be greatly appreciated. Thanks in advance!

**Comment by davevad93** [comment, marked as answer]
Hi @md-ashraful-islam-ullas ,

During the scheduling process you can choose if you want to take the exam in a local test center or online using your own computer.

For more information, consult the docs:

- [Registering for a GitHub Certifications exam](https://docs.github.com/en/get-started/showcase-your-expertise-with-github-certifications/registering-for-a-github-certifications-exam)
